modules/ROOT/pages/resource-management.adoc

```diff
@@ -30,6 +30,12 @@ contentType=layout
 
 If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.
 
+[NOTE]
+
+====
+To insert line breaks in a page.properties file, end the line that preceeds the line break with a "\".
```

Comment:
I suggest you ton point to the official properties javadoc si users will have the fully detailed explanation, with all the important things to take into account to create 'advanced' properties file. 
I suggest you to put it in a TIP admonition, with a content like 

"To write advanced properties file (line break, natural lines, logical lines...) please refer to the official documentation -> https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html#load-java.io.Reader-"

Reply:
@NathalieC What about this pull request? 
If you don't want to take it account suggestions let us know.

Reply:
@NathalieC  UUUUPP :)

Reply:
I'm cooooming.

Reply:
I've added what I think to be the best of both worlds: a quick tip for the use case we know is an issue (line breaks) + the official link to the documentation, to deepen one's knowledge on properties.